File: files/400-499/424/CH-424.R
library(tidyverse)
library(readxl)

path <- "400-499/424/CH-424 Replacement.xlsx"
input <- read_excel(path, range = "B3:D9")
test <- read_excel(path, range = "F3:H9")

result <- input %>%
  mutate(`Product ID` = chartr("ABC", "BCA", `Product ID`))

all.equal(result, test)
# True

File: files/400-499/424/CH-424.py
import pandas as pd

path = "400-499/424/CH-424 Replacement.xlsx"
input = pd.read_excel(path, usecols="B:D", nrows=7, skiprows=2)
test = pd.read_excel(path, usecols="F:H", nrows=7, skiprows=2)
test.columns = test.columns.str.replace(r"\.1$", "", regex=True)

result = input.copy()
result["Product ID"] = (
    result["Product ID"].astype(str).str.translate(str.maketrans("ABC", "BCA"))
)

print(result.equals(test))
# True
